keras-to-hls/keras-to-hls.py

```diff
@@ -191,8 +193,15 @@ def main():
                     else:
                         layer['n_subout'].append(layer['n_out']-n_totout)
                         n_totout += layer['n_out']-n_totout
-
-                    layer['n_part'] += 1   
+                    layer['n_part'] += 1
+                for i_part in range(0,layer['n_part']):
+                    i_subout = 0
+                    if i_part>0:
+                        i_subout = sum(layer['n_subout'][0:i_part])
+                    cur_n_zeros = print_array_to_cpp("w{}".format(layer_counter), weights, yamlConfig['OutputDir'], i_part, layer['n_part'], i_subout, layer['n_subout'][i_part])
+                    print_array_to_cpp("b{}".format(layer_counter), biases, yamlConfig['OutputDir'], i_part, layer['n_part'], i_subout, layer['n_subout'][i_part])
+                    layer['weights_n_subzeros'].append(cur_n_zeros)
+            
```

Comment:
Can this move to the central hls_writer.py? Then this can be picked up by Pytorch-to-hls.py as well